app/soundcloud.js

```diff
@@ -75,9 +76,11 @@ module.exports = class SoundCloud extends Events {
       titleParts = title.split(' in ', 2)
     if (titleParts.length == 2)
       // Title has " in " in it when not playing but on a playlis page
-      this.isPlaying().then(isPlaying => {
+      this.isPlaying().then(values => {
+        var isPlaying = values[0]
+        var icon = values[1]
```

Comment:
Assuming we switch to passing in an object as suggested above, there is an easier way of extracting the values:

```js
this.isPlaying().then({isPlaying, icon} => {
  if (isPlaying) {
    this.emit('play', titleParts[0], titleParts[1], icon)
  }
})
```